libmscore/cmd.cpp

```diff
@@ -1700,12 +1703,12 @@ void Score::cmdResetBeamMode()
       int endTick   = selection().tickEnd();
 
       Segment::Type st = Segment::Type::ChordRest;
-      for (Segment* seg = firstMeasure()->first(st); seg; seg = seg->next1(st)) {
+      for (Segment* seg = selection().startSegment(); seg; seg = seg->next1(st)) {
             if (seg->tick() < startTick)
```

Comment:
This check is probably no longer required, as you now start from selection start